app/components/formatted_time.js

```diff
@@ -3,35 +3,32 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import 'date-time-format-timezone/build/src/date-time-format-timezone-all-zones-no-locale';
+import {injectIntl, intlShape} from 'react-intl';
```

Comment:
you can remove the `injectIntl` and have the `intl` as part of the context instead of the props by doing:

```js
static contextTypes = {
    intl: intlShare.isRequired,
};
```

and then
```js
const {intl} = this.context;
```